packages/web/src/services/solana/solana.ts

```diff
@@ -47,3 +47,21 @@ export const getUSDCUserBank = async (ethAddress: string) => {
   })
   return usdcUserBank
 }
+
+/**
+ * Returns the current user's user bank for a specific token mint.
+ */
+export const getUserBank = async (ethAddress: string, mint: string) => {
+  const mintMap = {
+    AUDIO: 'wAUDIO',
+    USDC: 'USDC',
+    BONK: 'BONK'
+  } as const
```

Comment:
we shouldn't be hardcoding values here. we should just pass in the `mint` and in `deriveUserBank` do `mint: new PublicKey(mint)`